src/Core/Termination.idr

```diff
@@ -362,7 +363,7 @@ mutual
       = do Just gdef <- lookupCtxtExact fn_in (gamma defs)
                 | Nothing => throw (UndefinedName fc fn_in)
            let fn = fullname gdef
-           log "termination" 10 $ "Looking under " ++ show fn
+           log "termination.termination.sizechange" 10 $ "Looking under " ++ show fn
```

Comment:
`termination.termination`?

Reply:
i was carrying on from the existing topic! Maybe we should change the lead topic to be `totality`?

Reply:
I see. I think `totality` would indeed be clearer. Or just not having `termination` and
`positivity` under the same heading (we can always use multiple topics anyways).

Reply:
TBH I think what is also happening is that I assume that topics are presented hierarchically i.e. "grandparent.parent.child". This is probably where our mental models differ. Regardless the change has been made.